Article/Article.js

```diff
@@ -3,24 +3,32 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'expand';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => { 
+      this.expandArticle()
+    });
   }
 
   expandArticle() {
```

Comment:
Everything looks great but you could have used an if statement here in order to have the innertext to say close of open based on whether the article is opened or closed.